molgenis-jobs/src/main/java/org/molgenis/jobs/JobExecutionContextFactory.java

```diff
@@ -1,8 +1,12 @@
 package org.molgenis.jobs;
 
 import org.molgenis.jobs.model.JobExecution;
+import org.springframework.security.core.Authentication;
 
 public interface JobExecutionContextFactory {
 
   JobExecutionContext createJobExecutionContext(JobExecution jobExecution);
+
+  JobExecutionContext createJobExecutionContext(
```

Comment:
Let's give this one a different name, like createJobExecutionContextWithAuthentication or so